entries/c/calm.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Clam": {
+    "domain": "calm.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.calm.com/hc/articles/27928137411995",
```

Comment:
Calm has three different MFA systems for their customers, [partners](https://support.calm.com/hc/articles/27928137411995), and [health customers](https://www.calmhealth.com/mfa). I can't find their documentation for their customers, so could you please attach some screenshots of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

Reply:
I think we can temporarily close this PR. I tested my account and found that I couldn’t log in with my phone number (but it still needs a phone number), so I want to wait until the 2FA start process is displayed after the successful login before opening this PR. What do you think? @hkamran80